app/views/characters/_editor.haml

```diff
@@ -22,34 +22,35 @@
         = label_tag :new_template, 'Create New Template'
   - if @character.user == current_user
     %tr#create_template{class: ('hidden' unless params[:new_template])}
-      %th.sub &#8627; Name
-      %td{class: klass}
-        = f.fields_for :template do |ff|
+      = f.fields_for :template do |ff|
+        %th.sub= ff.label :name, '↳ Name'
+        %td{class: klass}
           = ff.text_field :name, placeholder: "Template Name", class: 'text'
+          = ff.hidden_field :id
```

Comment:
But this is guaranteed to be nil, isn't it? This is the Create New Template form?

Reply:
if you hit 'create instance' on a template, it will take you to this form, with 'id' in the subform. Don't ask me _why_ but that's the behaviour. So I uh... told it to put it here so it didn't try to do it in a way that generated invalid html.